README.md

```diff
@@ -39,24 +45,93 @@ If you intend to use the software constantly, you can install for the user or at
 Installation for the user can be carried out by:
 
 ```bash
-$ cp h1_client.bin ~/.local/bin/h1
+$ cp h1-linux ~/.local/bin/h1
+$ h1
+```
+
+Installation at the system level can be carried out by:
+
+```bash
+$ sudo cp h1-linux /usr/local/bin/h1
 $ h1
 ```
 
+### Alpine environment
+
+You need to install *libstdc++* package using apk:
+
+```bash
+$ sudo apk add libstdc++
+```
+
+Download the *h1-alpine.tar.gz* file from [Releases] using wget:
+
+```bash
+$ wget https://github.com/hyperonecom/h1-cli/releases/latest/download/h1-alpine.tar.gz
+```
+
+Then unpack the downloaded archive:
+
+```bash
+$ tar -xzvf h1-alpine.tar.gz
+$ ./h1-alpine
+```
+
+The application should work correctly.
+
+If you intend to use the software constantly, you can install at the system level.
+
 Installation at the system level can be carried out by:
 
 ```bash
-$ cp h1_client.bin /usr/local/bin/h1
+$ sudo cp h1-alpine /usr/local/bin/h1
```

Comment:
Dlaczego w archiwach mają być zamieszczone pliki z postfixami?

Reply:
A czy uważasz, że jest to utrudnienie dla użytkownika?

Reply:
To znaczy: zamieściłem plik z postfixem bo tak było łatwiej